packages/vscode-extension/src/qm/vsc_ui.ts

```diff
@@ -130,8 +132,18 @@ export class VsCodeUI implements UserInteraction {
     try {
       const quickPick = window.createQuickPick<FxQuickPickItem>();
       quickPick.title = option.title;
+      const buttons: QuickInputButton[] = option.buttons
+        ? option.buttons.map((button) => {
+            return {
+              iconPath: new ThemeIcon(button.icon),
```

Comment:
Will this support customized icon?

Reply:
I prefer to use customized icon in contribution point instead of icon path directly since passing icon path will not adjust to multiple themes.

Reply:
ok, looks good to me